odoo/addons/base/models/ir_mail_server.py

```diff
@@ -733,30 +734,41 @@ def _find_mail_server(self, email_from, mail_servers=None):
         if mail_servers is None:
             mail_servers = self.sudo().search([], order='sequence')
 
+        def _match_server_filter_email_from(mail_server, target_email):
```

Comment:
If we are going to create inline methods, why not let them take the "mail_servers" recordset and return the "next" that matches our condition?

Currently it seems like you always filter everything and yet only care about the first match.

(Also, are inline methods usually "private" starting with an underscore? Not sure it does anything).

Reply:
Perfect, can we just rename "process" to "normalize_method" ?
A bit more self-speaking.

After that we should be good to go :rocket:

Reply:
more explicit indeed :) done